crates/ruff/src/checkers/physical_lines.rs

```diff
@@ -118,7 +118,7 @@ pub fn check_physical_lines(
         }
 
         while doc_lines_iter
-            .next_if(|doc_line_start| line.range().contains(**doc_line_start))
+            .next_if(|doc_line_start| line.range().contains_inclusive(**doc_line_start))
```

Comment:
This was getting stuck on empty-lines-in-docstrings. I think `contains_inclusive` is correct here?